src/m_misc.c

```diff
@@ -463,6 +464,13 @@ default_t defaults[] = {
     "1 to enable \"ghost monsters\" (resurrected pools of gore are translucent)"
   },
 
+  {
+    "hangsolid",
+    (config_t *) &hangsolid, NULL,
+    {0}, {0,1}, number, ss_enem, wad_no,
```

Comment:
We have collected some unrelated config keys with `ss_enem` category again. Will have to clean this up (probably only after this PR, though).

Reply:
Oh, I see. I always forget about it.

Reply:
What would be the right category, btw?

Reply:
> What would be the right category, btw?

`ss_gen` for "General" menu.